src/content/en/2020/performance.md

```diff
@@ -228,58 +240,64 @@ First Contentful Paint (FCP) measures the first time at which the browser render
+Before we dig into the analysis, it is noteworthy to mention that in the 2019 Performance chapter, the thresholds for "good" and "poor" classification were different from 2020. In 2019, sites with FCP below 1s were considered good, while those with FCP above 3s were categorized as poor. In 2020, those ranges shifted to 1.5s for good and 2.5s for poor.
 
-While the mediocre (”needs improvement”) percentage becomes smaller, the number of poorly performing FCP sites rises, which is especially pronounced at the bottom of the ranking with LATAM and South Asian regions.
+This change means that the distribution would shift towards more "good" and "poor" rated websites. We can observe that trend compared to [last year's results](../2019/performance#fcp-by-geography), as the percentage of good and poor websites rise. The top ten geographies with the highest rate of fast websites remain relatively unchanged from 2019, with the addition of Czechia and Belgium and the fall of the United States and the United Kingdom. The Republic of Korea leads with 62% of websites reporting fast FCP, nearly doubling since last year (which, again, is likely due to re-categorization of results). Other countries in the top of the ranking also double the number of good experiences.
 
-Again, hypothesizing on reasons for the specific rankings is difficult, especially seeing outliers such as Australia at the lower end. Australia has one of the highest global smartphone penetration levels, one of the fastest mobile networks and relatively high average income levels. We’d easily assume it should be higher up. However, taking into account slow fixed connections, latency and lack of iOS representation in CrUX, its positioning starts making more sense. With an example like this (touched only on the surface), we can see how difficult understanding context for each of the countries would be.
+While the mediocre ("needs improvement") percentage becomes smaller, the number of poorly performing FCP sites rises, which is especially pronounced at the bottom of the ranking with Latin American and South Asian regions.
+
+Again, hypothesizing on reasons for the specific rankings is difficult {# TODO hypothesizing is (relatively) easy, proving is difficult #} }, especially seeing outliers such as Australia at the lower end. Australia has one of the highest global smartphone penetration levels, one of the fastest mobile networks and relatively high average income levels. We'd easily assume it should be higher up. However, taking into account slow fixed connections, latency and lack of iOS representation in CrUX, its positioning starts making more sense. With an example like this (touched only on the surface), we can see how difficult understanding context for each of the countries would be.
```

Comment:
@thefoxis thoughts?

Reply:
@bazzadp I don't see any reason to reword this to `While hypothesizing on reasons for specific rankings is easy, backing those speculations is difficult, especially seeing outliers such as Australia at the lower end.`

also, the reason there is probably the fact that iPhones aren’t counted, which is hands-down one of the most popular mobile devices in AU.

Reply:
Oh I wasn't suggesting rewording to that! Sorry that wasn't clear at all.

What I meant is you have good theories here and I feel you're doing yourself a disservice in both this and the last point by down playing this with the first phrase in these sections. The current wording (of both sections) reads like "We can't guess why this is. This is probably due to..." which kind of reads contradictory.

Instead of saying it's difficult to hypothesise about why this is (and then hypothesising about this), would it sound better to just say something like "There are a number of reasons why this could be, including..."?

Reply:
ahh, gotcha, I misunderstood. it's not really a writing style I have, I just was trying to mimic 2019 there :) we could rephrase this para to something along these lines:

> Again, there are several reasons negatively affecting FCP, such as poor TTFB readings, but it is difficult to prove them without the necessary context. For example, if we were to analyze specific country performance, such as Australia, we surprisingly find it at the lower end. Australia has one of the highest global smartphone penetration levels, one of the fastest mobile networks and relatively high average income levels. We'd easily assume it should be higher up. However, taking into account slow fixed connections, latency and lack of iOS representation in CrUX, its positioning starts making more sense. With an example like this (touched only on the surface), we can see how difficult understanding context for each of the countries would be.

but I'm also happy to remove those couple sentences, as at this point I'm starting to doubt how useful it is 😛

Reply:
Thanks. Changed.